README.md

```diff
@@ -74,8 +72,10 @@ composer require rector/rector --dev
 **Did you have conflicts during `composer require` or on run?**
 - Use the [Docker image](#run-rector-in-docker)
 
-## Running Rector
+<br>
+<br>
 
+## Running Rector
```

Comment:
It's convention to separate markdown headlines with spaces, so the text is more readable and clear, that there is huge space following (bigger than between text liens)